autonomy/cli/mint.py

```diff
@@ -217,3 +217,62 @@ def agent(
         nft_image_hash=nft,
         skip_hash_check=ctx.config.get("skip_hash_check", False),
     )
+
+
+@mint.command()
+@package_path_decorator
+@key_path_decorator
+@password_decorator
+@nft_decorator
+@pass_ctx
+@click.option(
+    "-a",
+    "--agent-id",
+    type=int,
+    help="Canonical agent ID",
+    required=True,
+)
+@click.option(
+    "-n",
+    "--number-of-slots",
+    type=int,
+    help="Number of agent instances for the agent",
+    required=True,
+)
+@click.option(
+    "-c",
+    "--cost-of-bond",
+    type=int,
+    help="Cost of bond for the agent (Wei)",
+    required=True,
+)
+@click.option(
+    "--threshold",
+    type=int,
+    required=True,
+)
+def service(  # pylint: disable=too-many-arguments
```

Comment:
To mint a service run

`autonomy mint service SERVICE_PATH KEY_PATH -a AGENT_ID -n NUMBER_OF_SLOTS -c COST_OF_BOND --threshold THRESHOLD`

Although we support multiple agents per service on the smart contract we don't have that functionality available on the framework side yet so for now we'll only allow one agent per service